tests/tests/voting_tests.cpp

```diff
@@ -393,4 +393,26 @@ BOOST_AUTO_TEST_CASE(track_votes_committee_disabled)
    } FC_LOG_AND_RETHROW()
 }
 
+BOOST_AUTO_TEST_CASE(invalid_voting_account)
+{
+   try
+   {
+      ACTORS((alice));
+
+      account_id_type invalid_account_id( (uint64_t)999999 );
+
+      BOOST_CHECK( !db.find( invalid_account_id ) );
+
+      graphene::chain::account_update_operation op;
```

Comment:
Do we have a test for account_create_operation too?

Reply:
https://github.com/bitshares/bitshares-core/blob/2d7bfd8bdefb7951abe8f5dfca386c3da16e1616/tests/tests/operation_tests.cpp#L882